packages/javascript-sdk/src/http-client/index.ts

```diff
@@ -28,11 +28,13 @@ import {
   addAuthzInfoToURL,
   buildAuthzOptions,
   examineForIGAuthz,
+  examineForIGAuthzHeader,
   examineForRESTAuthz,
   hasAuthzAdvice,
   isAuthzStep,
   newTokenRequired,
```

Comment:
This import of newTokenRequired (and the subsequent usage), was removed in the PoC code. @cerebrl can you remember why this was? I kept it in as I couldn't see how this affects the flow.

Reply:
I commented it out, but I don't remember why. I was likely just for ease of testing; definitely bring it back :)

Reply:
Ok no worries, I have kept it in there guessing this might be the case.